stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -69,7 +143,185 @@ private static String getIdAttribute(View element) {
 
   @Override
   public View getViewForHighlighting(Object element) {
-    return (View)element;
+    return (View) element;
+  }
+
+  @Override
+  protected void onGetStyles(View element, StyleAccumulator styles) {
+
+    for (ViewCSSProperty property : getProperties()) {
+      Object value = null;
+      boolean caught = false;
+
+      try {
+        value = property.getValue(element);
+      } catch (IllegalAccessException e) {
+        caught = true;
+      } catch (InvocationTargetException e) {
+        caught = true;
+      } finally {
+        if (caught) {
+          LogUtil.e("failed to get the style of element id=" + getIdAttribute(element));
```

Comment:
Instead of using a flag to determine if we caught an exception and then logging it, try doing the logging from one catch clause, like so

```
try {
  value = property.getValue(element);
} catch (IllegalAccessException | InvocationTargetException ex) {
  value = null; // this is where'd you set it to null, re: my previous comment
  LogUtil.e(ex, ...);
}
```